Gemfile

```diff
@@ -8,6 +8,7 @@ gemspec
 # jquery-rails is used by the dummy application
 gem "jquery-rails"
 gem "mysql2"
+# gem "postgresql_cursor"
```

Comment:
Oh I'll take it out. I was trying something and then must've forgotten to delete instead of comment it out.